Index: random/0. Ideas.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>Initiating ideas:\r\n1. creat time table\r\n2. import time to notify pre-set scheadule\r\n3.import voices\r\n6.use and stydy and creat algorytm which creace data, ask user feedback, input user responses, train new module and create new data. Just like online AI does\r\n9. cooking book. make cooking section with exotic materials\r\n11. scrap WSJ\r\n\r\n\r\n\r\nCooking links:\r\nhoney chicken, apple cider, sear brocoli\r\n1. https://www.youtube.com/watch?v=3SqoPSO_W9U\r\nteri chicken garlic butter udon\r\n2. https://www.youtube.com/watch?v=zkBEHKAl0Kk\r\n\r\nneed white pepper powder, mustard, unsalted butter
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/random/0. Ideas.txt b/random/0. Ideas.txt
--- a/random/0. Ideas.txt	(revision 191327da2493d3e8941df76ebe5e82864c4489e8)
+++ b/random/0. Ideas.txt	(date 1737167080819)
@@ -5,7 +5,8 @@
 6.use and stydy and creat algorytm which creace data, ask user feedback, input user responses, train new module and create new data. Just like online AI does
 9. cooking book. make cooking section with exotic materials
 11. scrap WSJ
-
+12.continue work with food.csv in data folder
+13.continue on file no 8 and 9
 
 
 Cooking links:
Index: random/Data/Food.csv
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/random/Data/Food.csv b/random/Data/Food.csv
new file mode 100644
--- /dev/null	(date 1737167080820)
+++ b/random/Data/Food.csv	(date 1737167080820)
@@ -0,0 +1,2 @@
+No.,Title,Ingredients,Process
+1,,,pass
\ No newline at end of file
